README.md

```diff
@@ -267,13 +253,6 @@ I would like to thank...
 
 > Add at least 2 questions new developers would ask when they decide to use your project.
 
```

Comment:
- [ ] As suggested by the previous reviewer, this message is for you to know how to use the README template not to leave it the users of your project. Kindly remove it and any other ones of its type in the README. They are there to guide you. After implementing the message those texts pass across to you, you're expected to remove them. 

![readme-1](https://user-images.githubusercontent.com/26597432/219014773-722e0b72-970f-40f9-9de9-5a027e63b93c.png)


![readme-2](https://user-images.githubusercontent.com/26597432/219014812-6fa58d91-4ab0-48bb-9504-1cca20d08466.png)